integration/library_test.go

```diff
@@ -44,6 +44,7 @@ import (
 type testCase struct {
 	name                 string
 	imageName            string
+	dockerlessImageName  string
```

Comment:
This probably feels clunky but I thought about reusing imageName as the image name for docker less mode tests. It was even worse since we had to add extra test logic to massage it. I like this better, even though its slightly repetitive.